relnotes/verify.feature.md

```diff
@@ -0,0 +1,9 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to [Sociomantic assert/enforce
+  policies](https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/assert_vs_enforce.md)
```

Comment:
:+1: We can discuss a better structure. The main thing is that we have it in tsunami now.